### ISSUE TITLE
Add image button should be a regular button

### ISSUE THREAD

**Issue by jandubois** [rancher-sandbox/rancher-desktop#3478]
Right now the Images page has a big `+` sign in a blue circle in the top right corner to "Add an image".

This is accessibility issues, so would need at least to be labeled. But it may be simpler (and easier to understand for the casual user) if we used a regular button with text.

Implement a secondary call-to-action button called "Add Image" (no icon).
